packages/regl-worldview/src/stories/GLText.stories.js

```diff
@@ -86,5 +86,51 @@ storiesOf("Worldview/GLText", module)
         </Container>
       );
     }
+    return <Example />;
+  })
+  .add("highlighted text", () => {
+    const Example = () => {
+      const [searchText, setSearchText] = useState("ello\nW");
+      const markers = textMarkers({ text: "Hello\nWorldview" }).map((marker) => {
+        if (!searchText) {
+          return marker;
+        }
+        const highlightedIndices = new Set();
+        let match;
+        let regex;
+        try {
+          regex = new RegExp(searchText, "ig");
+        } catch (e) {
+          return marker;
+        }
+        while ((match = regex.exec(marker.text)) !== null) {
+          Array.from(Array(searchText.length).keys()).forEach((i) => {
+            // $FlowFixMe - Flow doesn't understand the while terminating condition.
+            highlightedIndices.add(match.index + i);
+          });
+        }
+        return { ...marker, highlightedIndices: Array.from(highlightedIndices) };
+      });
+
+      return (
+        <>
+          <div style={{ width: "100%", height: "95%" }}>
+            <Container cameraState={{ perspective: true, distance: 40 }} backgroundColor={[0.2, 0.2, 0.4, 1]}>
+              <GLText autoBackgroundColor>{markers}</GLText>
```

Comment:
Might be nice to have a demo without autoBackgroundColor to make sure it works. Maybe having both is best.

Reply:
will do